app/src/components/ui/Button.tsx

```diff
@@ -24,6 +24,10 @@ const buttonStyles = cva(
       fullWidth: {
         true: "w-full",
       },
+      design: {
+        // for icon only buttons (icon has to be square shaped)
+        circular: "p-2.5 rounded-full",
```

Comment:
oh `p-2.5` works?

Reply:
Ye it's should be 10px according to tailwind and works for me. I referenced Lindsey's figma design cuz the padding was 10px

Reply:
<img width="379" alt="image" src="https://github.com/lsuan/link-up/assets/39757033/2ab45471-9d12-4e25-858f-0017f2083ee3">
lmk if you want me to change it, but p-2.5 does give 10px padding

Reply:
ooo okay, this is fine !